.github/workflows/cpu_ci.yml

```diff
@@ -0,0 +1,42 @@
+name: "Run CPU Tests"
```

Comment:
I've read that Actions has `composites` where we can make this more DRY. I don't think it's currently necessary with how simple this pipeline is.